static/src/stylesheets/module/onward-garnett/_feedback.scss

```diff
@@ -48,6 +48,33 @@
     width: 80%;
 }
 
+.feedback__alertbox {
+    background-color: $news-garnett-main-1;
```

Comment:
I think this variable has been renamed `$news-main` following @zeftilldeath's PR https://github.com/guardian/frontend/pull/19968